core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -274,7 +274,7 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
           worker.stop()
         } catch {
           case e: Exception =>
-            logWarning("Failed to stop worker")
+            logWarning("Failed to stop worker", e)
```

Comment:
Shall we use `log" ... "`, even though there is no `MDC` embedded?
```suggestion
            logWarning(log"Failed to stop worker", e)
```

Reply:
done